gammapy/datasets/map.py

```diff
@@ -287,6 +287,11 @@ def excess(self):
         """Excess"""
         return self.counts - self.background
 
+    @property
```

Comment:
Is this really useful? I find it a bit confusing to have `aeff` on a `MapDataset`

Reply:
I though it was maybe useful, but I'm fine to remove it again...